posix.mak

```diff
@@ -455,7 +455,7 @@ rsync : all kindle pdf
 rsync-only :
 	rsync -avzO --chmod=u=rwX,g=rwX,o=rX --delete $(RSYNC_FILTER) $W/ $(REMOTE_DIR)/
 
-dautotest: all verbatim pdf diffable-intermediaries
+dautotest: all verbatim pdf diffable-intermediaries d-latest.tag d-prerelease.tag
```

Comment:
I will make a quick follow-up which removes `--only-tags` from `d-latest.tag`

Reply:
Whoops, right - with `--only-tags`, we don't actually pull in and test `chm-nav-*.json`. Well spotted.